tigerpath/majors_and_certificates/majors/EEB_2018.json

```diff
@@ -97,113 +124,117 @@
           "completed_by_semester": 7,
           "course_list": [
             "SML 201",
-            "ECO 202",
             "ORF 245",
-            "POL 345",
             "PSY 251",
+            "POL 345",
+            "SOC 301",
+            "ECO 202",
+            "WWS 220",
             "WWS 332"

```

Comment:
And then this course hasn't been offered in ages, but haha I guess it's fine to leave it in in case it reappears somehow.